linux/debian/control

```diff
@@ -85,7 +85,7 @@ Section: python
 Architecture: all
 Depends:
  dconf-cli,
- gir1.2-webkit2-4.0,
+ gir1.2-webkit2-4.1 | gir1.2-webkit2-4.0,
```

Comment:
For the Debian Unstable package, I recommend dropping the alternate dependency. The 4.0 package soon won't get security support in Debian Testing and the alternate dependency allows people to keep it installed.

Reply:
We still build packages for Ubuntu 20.04 which unfortunately doesn't have webkit2 4.1.